db/db.go

```diff
@@ -113,6 +113,9 @@ func majorDotMinor(osVer string) (majorMinorVersion string) {
 // getAmazonLinux2 returns AmazonLinux1 or 2
 func getAmazonLinux1or2(osVersion string) string {
```

Comment:
Since Amazon Linux 22 is also detected in addition to Amazon Linux 1 and 2, it would be better to change the name of this function.

Reply:
done